src/experimental/patterns/ec2-app.ts

```diff
@@ -251,6 +253,19 @@ export interface GuEc2AppExperimentalProps extends Omit<GuEc2AppProps, "updatePo
    * process.env.GITHUB_RUN_NUMBER
    */
   buildIdentifier: string;
+  /**
+   * The amount of time that newly launched instances will spend warming-up / serving fewer requests than other instances
+   * (defaults to 0 seconds).
+   *
+   * See https://docs.aws.amazon.com/elasticloadbalancing/latest/application/edit-target-group-attributes.html#slow-start-mode
+   * for more details.
+   *
+   * We recommend enabling this setting if you run a high-traffic service, particularly if it is JVM-based.
+   *
```

Comment:
Are we able to recommend a value? Or how to calculate a value? For example, would it be always be greater than one of the configurable healthcheck values?

Alternatively, could we log a warning/error if the duration is something unadvisable, like 1 hour?

Reply:
Unfortunately I think it will depend on the app and will likely need performance testing to optimise this effectively. 

For now I've just copied over some of the [AWS docs](https://docs.aws.amazon.com/AWSCloudFormation/latest/UserGuide/aws-properties-elasticloadbalancingv2-targetgroup-targetgroupattribute.html) and added some validation that was missing before: https://github.com/guardian/cdk/pull/2660/commits/228d7dafda503e2c6b7c2b19a0967ec677eebdc5.